engines/docker/generated_configschema.go

```diff
@@ -0,0 +1,47 @@
+// This source code file is AUTO-GENERATED by github.com/taskcluster/jsonschema2go
+
+package docker
+
+import "github.com/taskcluster/taskcluster-worker/runtime"
+
+type (
+	// Config applicable to docker engine
+	Config struct {
+
+		// Root Volume blah blah
+		RootVolume string `json:"rootVolume"`
+	}
+)
+
+func ConfigSchema() runtime.CompositeSchema {
```

Comment:
>  Or would something call that function at startup?

The pattern I wrote here:
`var configSchema = func() runtime.CompositeSchema { .... return ... ; }()`
Notice the subtle `}()` that the end... I'm actually setting `configSchema` to be the result of an anonymous function that is immediately invoked :)

So it'll be executed before `main()` is called.